client/src/components/ReviewStarsByDescriptor.jsx

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import StarsByDescriptor from './StarsByDescriptor';
+
+import {
+  Wrapper,
+  ColumnWrapper,
+  DescriptorWrapper,
+  DescriptorText,
+} from './styles/ReviewStarsByDescriptor.styles';
+
 const ReviewStarsByDescriptor = props => (
-  <div>
-    <h6> Accuracy {props.accuracyAvg} </h6>
-    <h6> Communication {props.communicationAvg} </h6>
-    <h6> Cleanliness {props.cleanlinessAvg} </h6>
-    <h6> Location {props.locationAvg} </h6>
-    <h6> Check-in {props.checkinAvg} </h6>
-    <h6> Value {props.valueAvg} </h6>
-  </div>
+  <Wrapper>
+    <ColumnWrapper>
+      <DescriptorWrapper>
```

Comment:
You could map all of these instead of creating new <DescriptorWrapper>s for all. You can place the text in an array/obj, and map those with the corresponding stars. You'd only have like probs 20 lines of code (or shorter) for this all.

Reply:
Totally right!  Great point, I will try it out.